cmd/buildah/manifest.go

```diff
@@ -226,6 +227,9 @@ func init() {
 	flags.StringVar(&manifestPushOpts.certDir, "cert-dir", "", "use certificates at the specified path to access the registry")
 	flags.StringVar(&manifestPushOpts.creds, "creds", "", "use `[username[:password]]` for accessing the registry")
 	flags.StringVar(&manifestPushOpts.digestfile, "digestfile", "", "after copying the image, write the digest of the resulting digest to the file")
+	flags.BoolVarP(&manifestPushOpts.forceCompressionFormat, "force-compression", "", false, "force re-compressing layers with specified format while pushing")
```

Comment:
In both cases: The option does not actually force re-compression; as the new description in `PushOptions` says, it prevents reuse of layers compressed with other algorithms.

“use the specified compression algorithm if the destination contains a differently-compressed variant already”? That’s a mouthful…

Reply:
Done.